Calculator/Calculator.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+public class Calculator {
+    public static void main(String[] args) {
+        double firstNumber = 0.0;
+        double secondNumber = 0.0;
+        char operator = '+';
```

Comment:
Please remove + from initialization of variable. It should not be initialized before user input (or leave blank)